k8s/service_controller/model/service_instance.go

```diff
@@ -22,9 +22,9 @@ type LastOperation struct {
 }
 
 type CreateServiceInstanceRequest struct {
```

Comment:
I think we need to have two different models. This one was written with the API to the SB in mind, which isn't the same as the API to the CC.  Your edits are for the CC API.

Perhaps we should prefix these with `SB_` and prefix the CC ones with `SC_` ??